requirements.txt

```diff
@@ -4,3 +4,4 @@ python_dotenv
 pymongo
 mcstatus
 aiohttp[speedups]
+flask
```

Comment:
Just a heads up for the future, flask and discord.py aren't able to run simultaneously and quart should be used instead

Reply:
updated